### PR TITLE
Update and rename website/partners/solutions to website/partners/solutio...

### PR DESCRIPTION
**Pull request by WernerDH**
...ns/conversionlift/index.hbs

This is the partner directory entry for Conversionlift

@juliritchi: Hi Julie, please review this pull request.

### CODE REVIEW

**Comment by WernerDH** [comment]
@julieritchie :  Hi Julie, this is the pull request for the PD entry of ConversionLift. Please let me know if I need to make corrections or give additional info.

File: website/partners/solutions/conversionlift/index.hbs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: partners.hbs
+title: "ConversionLift"
+solutions: true
+priority: 50
+summary:
+logo: "//optimizely-marketer-assets.s3.amazonaws.com/partners/logos/conversionlift.png"
+website_link: "http://www.conversionlift.de/optimizely-solutions-partner"
+website_display: "http://www.conversionlift.de"
+kb_article:
```
Comment (dtothefp) [inline website/partners/solutions/conversionlift/index.hbs]: @WernerDH please make sure for `website_display` you omit the `http://`

File: website/partners/solutions/conversionlift/index.hbs [hunk 2]
```diff
@@ -0,0 +1,35 @@
+---
+layout: partners.hbs
+title: "ConversionLift"
+solutions: true
+priority: 50
+summary:
+logo: "//optimizely-marketer-assets.s3.amazonaws.com/partners/logos/conversionlift.png"
+website_link: "http://www.conversionlift.de/optimizely-solutions-partner"
```
Comment (dtothefp) [inline website/partners/solutions/conversionlift/index.hbs]: @WernerDH please check this path to the image. It is not correct I don't think. The link path will be something like this `https://d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/partner-logos/solutions/<filename>` but you will have to look in Cloud Cannon and copy the url to be sure. Also, please download the optimizely chrome extension which allows you to view the staging version of this branch so you can see that everything worked properly. @julieritchie could you please walk @WernerDH through this if he is unsure?

https://docs.google.com/a/optimizely.com/document/d/1JmR3PtoiSyxbAxUY6PvP3Z9jv2qy3_MzDz-31zZicAQ/edit?usp=sharing

File: website/partners/solutions/conversionlift/index.hbs [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+layout: partners.hbs
+title: "ConversionLift"
+solutions: true
+priority: 50
+summary:
+logo: "https://d1qmdf3vop2l07.cloudfront.net/optimizely-marketer-assets.cloudvent.net/raw/partner-logos/solutions/conversionlift.png"
```
Comment (stewsmith) [inline website/partners/solutions/conversionlift/index.hbs]: Please remove the `https:` here.

File: website/partners/solutions/conversionlift/index.hbs [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+website_link: "http://www.conversionlift.de/optimizely-solutions-partner"
+website_display: "www.conversionlift.de"
+kb_article:
+stars: 1
+industry: "Conversion Optimisation, Landing Page Optimisation"
+locations:
+- location:
+    phone: "+4930340603500"
```
Comment (stewsmith) [inline website/partners/solutions/conversionlift/index.hbs]: Is a spaced out phone number better? Like in Catbird Seat?

File: website/partners/solutions/schaetzcro/index.hbs [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+- "Danish"
+tags:
+- "E-Commerce / Retail" 
+- "Lead gen" 
+- "Finance" 
+- "Non-Profit" 
+- "Telecommunications"
+- "Media"
+---
+schaetz cro is a Swiss based agency specialized in Web Analytics,  Conversion Rate Optimization and Strategic Consulting. We are working for organizations that are recognized brand leaders in their fields such as Migros, die Post, Search.ch, Tamedia, Aduno Group, upc cablecom and Amnesty International.
```
Comment (stewsmith) [inline website/partners/solutions/schaetzcro/index.hbs]: Either "SchaetzCRO" or "schaetz cro" should be used. Should be consistent with the title.

**Comment by stewsmith** [comment]
LGTM after comments are addressed.

**Comment by WernerDH** [comment]
@stewsmith  hi stewart, I made the changes you have adressed:
- phone numbers spaced out (mindberry, conversionlift)
- harmonize "SchaetzCRO"

**Pull request merged**